components/org.wso2.carbon.identity.user.endpoint/src/gen/java/org/wso2/carbon/identity/user/endpoint/dto/SuccessfulUserCreationDTO.java

```diff
@@ -1,95 +1,86 @@
+    private String notificationChannel = null;
 
-@ApiModel(description = "")
-public class SuccessfulUserCreationDTO  {
-  
-  
-  
-  private String code = null;
-  
-  
-  private String message = null;
-  
-  
-  private String notificationChannel = null;
-  
-  
-  private String confirmationCode = null;
-
-  
-  /**
-   * Status code of the operation.
-   **/
-  @ApiModelProperty(value = "Status code of the operation.")
-  @JsonProperty("code")
-  public String getCode() {
-    return code;
-  }
-  public void setCode(String code) {
-    this.code = code;
-  }
-
-  
-  /**
-   * Descriptive message regarding the operation.
-   **/
-  @ApiModelProperty(value = "Descriptive message regarding the operation.")
-  @JsonProperty("message")
-  public String getMessage() {
-    return message;
-  }
-  public void setMessage(String message) {
-    this.message = message;
-  }
+    /**
+    * Status code of the operation.
+    **/
+    @ApiModelProperty(value = "Status code of the operation.")
+    @JsonProperty("code")
+    public String getCode() {
+        return code;
+    }
+    public void setCode(String code) {
+        this.code = code;
+    }
 
-  
-  /**
-   * Account confirmation notification sent channel.
-   **/
-  @ApiModelProperty(value = "Account confirmation notification sent channel.")
-  @JsonProperty("notificationChannel")
-  public String getNotificationChannel() {
-    return notificationChannel;
-  }
-  public void setNotificationChannel(String notificationChannel) {
-    this.notificationChannel = notificationChannel;
-  }
+    /**
+    * Descriptive message regarding the operation.
+    **/
+    @ApiModelProperty(value = "Descriptive message regarding the operation.")
+    @JsonProperty("message")
+    public String getMessage() {
+        return message;
+    }
+    public void setMessage(String message) {
+        this.message = message;
+    }
 
-  
-  /**
-   * Confirmation code to verify the user, when notifications are externally managed. In this scenario, *notificationChannel* will be *EXTERNAL*. Use the confirmation code for confirm the user accounut.
-   **/
-  @ApiModelProperty(value = "Confirmation code to verify the user, when notifications are externally managed. In this scenario, *notificationChannel* will be *EXTERNAL*. Use the confirmation code for confirm the user accounut.")
-  @JsonProperty("confirmationCode")
-  public String getConfirmationCode() {
-    return confirmationCode;
-  }
-  public void setConfirmationCode(String confirmationCode) {
-    this.confirmationCode = confirmationCode;
-  }
+    /**
+    * Account confirmation notification sent channel.
+    **/
+    @ApiModelProperty(value = "Account confirmation notification sent channel.")
+    @JsonProperty("notificationChannel")
+    public String getNotificationChannel() {
+        return notificationChannel;
+    }
+    public void setNotificationChannel(String notificationChannel) {
+        this.notificationChannel = notificationChannel;
```

Comment:
Need new line before method signature, please check the other methods too

Reply:
These are auto-generated classes. It's better if we did not change them.